infrastructure/src/main/java/org/corfudb/infrastructure/BatchWriter.java

```diff
@@ -30,9 +31,10 @@
 @Slf4j
 public class BatchWriter<K, V> implements CacheWriter<K, V>, AutoCloseable {
 
+    // TODO(Maithem) make this dynamic
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)